modules/core/client/app/app.js

```diff
@@ -20,10 +20,44 @@ tropicalbs.config(function ($stateProvider, $urlRouterProvider) {
       abstract: true,
       controller: 'AppController'
     });
-});
 
-tropicalbs.config(function ($mdThemingProvider) {
+  $httpProvider.interceptors.push('AttachTokens');
+
+}])
+.factory('AttachTokens', ['$window', '$cookies', function ($window, $cookies) {
```

Comment:
fine for now but a lot of the things (config, factory for authentication) in this file should be split into a separate files